Atomic/install.py

```diff
@@ -59,6 +59,9 @@ def cli(subparser):
                 system_xor_user.add_argument("--system", dest="system",
                                              action='store_true', default=False,
                                              help=_('install a system container'))
+                installp.add_argument("---generate-rpm", dest="generate_rpm",
```

Comment:
the --generate-rpm doesn't probably belong here, I just  didn't know where to add it.

I think we will need a way to just generate the .rpm, so that it can be used as any regular .rpm, you can even install it without using atomic at all.

Installing the .rpm could be the default here, but we will need a way to specify also to install the container in the usual way using `/var/lib` instead of '/usr/lib' if needed.